pandas/util/mixins.py

```diff
@@ -0,0 +1,28 @@
+"""
+Mixins which can't go to core.common or core.base due to circular imports
+"""
```

Comment:
It can't because that would not work with `str = AccessorProperty(StringMethods, _make_str_accessor)` -> The `AccessorProperty` needs a handle to the real value to do this:

```
    def __get__(self, instance, owner=None):
        if instance is None:
            # this ensures that Series.str.<method> is well defined
            return self.accessor_cls
        return self.construct_accessor(instance)
```